spec/models/concerns/courses/edit_options/program_type_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe Courses::EditOptions::ProgramTypeConcern do
+  let(:example_model) do
+    klass = Class.new do
+      include Courses::EditOptions::ProgramTypeConcern
+
+      def self_accredited?; end
```

Comment:
If you set attributes directly and implement these methods - I think they'd be clearer to another reader as to what they're for. I find these pretty difficult to parse at first glance to understand why you'd have blank methods here

Reply:
Done.